src/System.Memory/tests/Performance/Perf.Utf8Parser.cs

```diff
@@ -10,12 +11,984 @@ namespace System.Buffers.Text.Tests
 {
     public static partial class Utf8ParserTests
     {
-        private const int InnerCount = 100000;
+        private static readonly string[] s_UInt32TextArray = new string[10]
+        {
+            "42",
+            "429496",
+            "429496729",
+            "42949",
+            "4",
+            "42949672",
+            "4294",
+            "429",
+            "4294967295",
+            "4294967"
+        };
+
+        private static readonly string[] s_UInt32TextArrayHex = new string[8]
+        {
+            "A2",
+            "A29496",
+            "A2949",
+            "A",
+            "A2949672",
+            "A294",
+            "A29",
+            "A294967"
+        };
+
+        private static readonly string[] s_Int16TextArray = new string[13]
+        {
+            "21474",
+            "2",
+            "-21474",
+            "31484",
+            "-21",
+            "-2",
+            "214",
+            "2147",
+            "-2147",
+            "-9345",
+            "9345",
+            "1000",
+            "-214"
+        };
+
+        private static readonly string[] s_Int32TextArray = new string[20]
+        {
+            "214748364",
+            "2",
+            "21474836",
+            "-21474",
+            "21474",
+            "-21",
+            "-2",
+            "214",
+            "-21474836",
+            "-214748364",
+            "2147",
+            "-2147",
+            "-214748",
+            "-2147483",
+            "214748",
+            "-2147483648",
+            "2147483647",
+            "21",
+            "2147483",
+            "-214"
+        };
+
+        private static readonly string[] s_SByteTextArray = new string[17]
+        {
+            "95",
+            "2",
+            "112",
+            "-112",
+            "-21",
+            "-2",
+            "114",
+            "-114",
+            "-124",
+            "117",
+            "-117",
+            "-14",
+            "14",
+            "74",
+            "21",
+            "83",
+            "-127"
+        };
+
+        [Benchmark]
+        [InlineData("2134567890")] // standard parse
+        [InlineData("18446744073709551615")] // max value
+        [InlineData("0")] // min value
+        private static void BaselineSimpleByteStarToUInt64(string text)
```

Comment:
The test name doesn't match what its doing and should be updated. This isn't converting byte* to ulong.

Maybe, `BaselineStringToUInt64` to compare against `ByteSpanToUInt64`